community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,396 @@
+GDScript, opening and closing menus etc.). Also, while reviewing the code, look
+for suspicious changes in other parts of the engine. Sometimes during rebasing
+changes slip through that contributors are not aware of.
+
+3. Communicate barriers to merging early in the review process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If something has clear blockers or will likely not get merged for whatever other
+reason, that fact should be communicated as early and clearly as possible. We
+want to avoid stringing people along because it feels bad to say "sorry, no".
+
+4. Do a code review
+^^^^^^^^^^^^^^^^^^^
+
+Code reviews are usually done by people who are already experienced in a given
+area. They may be able to provide ideas to make code faster, more organized, or
+more idiomatic. But, even if you are not very experienced, you may want to
+conduct a code review to provide feedback within the scope of what you are
+comfortable reviewing. Doing so is valuable for the reviewer (as a second set of
+eyes on a problem is always helpful) and it is also helpful for you as it will
+help you get more familiar with that area of code and will expose you to how
+other people solve problems. In fact, reviewing the code of experienced engine
+maintainers is a great way to get to know the codebase.
+
+Here are some things to think about and look out for as you review the code:
+
+* **Code only touches the areas announced in the PR (and the commit
+  message).**
+
+  It can be tempting to fix random things in the code, as you see them. However,
+  this can quickly make a pull request difficult to review and can make it hard
+  to dig through in the commit history. Small touch-ups next to the related area
+  are alright, but often bugs that you can find along the way are better fixed
+  in their own PRs.
+
+* **Code properly uses Godot's own APIs and patterns.**
+
+  Consistency is very important, and a solution that already exists in the
+  codebase is preferable to an ad-hoc solution.
+
+* **Are core areas affected by the change?**
+
+  Sometimes a PR that is supposed to solve a local problem can have a
+  far-reaching effect way outside of its scope. Usually it is best to keep code
+  changes local to where the problem arises. If you think that the solution
+  requires changes outside the scope of the problem, it is usually best to seek
+  the opinion of a team leader who may have another idea for how to solve the
+  problem.
+
+5. Iterate with the contributor and improve the PR
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Maintainers should provide feedback and suggestions for improvement if they spot
+things in the code that they would like changed. Preferably, suggestions should
+come in order of importance: first, address overall code design and the approach
+to solving the problem, then make sure the code is complying with the engine's
+best practices, and lastly, do the code style review (discussed in more detail
+below).
```

Comment:
Should link directly to the section, I think. You'd need to add `.. _doc_something:` before the section title to be able to link to it.